src/content/docs/products/ontrack/start-contributing.md

```diff
@@ -0,0 +1,101 @@
+
+Then without closing the remote connection to the container, rebuild the container using the Command Palette command (ctrl / cmd + shift + p) ```Dev Containers: Rebuild Container```. 
+
+##### 2. Docker container related errors
+Sometimes if you are having any weird Docker container related problems such as shown below, you might need to delete all **Containers, Images, and Volumes** related to **doubtfire-lms/formatif** from inside the **Docker app** and restart the **Docker Engine**.
+
+![Docker Error](/ontrack/contributing-guide/dockererror.png)
+
+After you have restarted the docker engine, you can then open the **doubtfire-deploy** folder in VS Code using the Command Palette command Dev Containers: Open Folder in Container... 
+
+##### 3. Frontend not starting
+Typically, if the frontend fails to start it means that you are missing the required packages. This can be resolved by opening a new terminal in the **doubtfire-web** directory and running the command ```npm install -f```. Once this command has finished running, run the command ```npm start``` to start the frontend, which can usually be accessed at http://localhost:4200/.
+
+##### 4. Wrong sign in page / frontend errors
+Errors that are encountered in the frontend typically show up as alerts that appear at the top right of the page.
+
+![Incorrect sign in page](/ontrack/contributing-guide/incorrectsigninpage.png)
+
+Most of the time these will occur due to the backend not working as a result of a migration error, which an example of is shown below:
+
+![Backend migration error](/ontrack/contributing-guide/backendmigrationerror.png)
+
+To fix this, enter the backend terminal and press **ctrl + c** to end the backend process. Once this process has ended, create a new terminal in the doubtfire-api directory. Run the following commands to fix the migration error and relaunch the backend:
+
+1. ```bundle exec rake db:migrate```
+2. ```bundle exec rake db:populate```
+3. ```rails s```
+
+Reload the frontend once the backend server is up and running. The correct sign-in page like shown below should appear.
+
+![Correct sign in page](/ontrack/contributing-guide/correctsigninpage.png)
+
+Login with username **student_1** and password **password** and enter **1** as the Student ID/number and sign-in. The dashboard should appear with the enrolled units.
+
+### Documentation and Templates
+
+#### Documentation Repositories
+Currently, ThothTech has 2 documentation repositories, [ThothTech/documentation](https://github.com/thoth-tech/documentation) and [ThothTech/ThothTech-Documentation-Website](https://github.com/thoth-tech/ThothTech-Documentation-Website), which is the repo site is located on. The ThothTech Documentation Website is the most current repository, which is being actively worked on and where all new documentation should be placed.
```

Comment:
This part needs to be changed. documentation is for internal docs, anything we can't release to the public like docs about new features we are working on. ThothTech-Documentation-Website is the external docs repo, anything we have that helps introduce new members into the company and products